ShopLens/ImageRecognition/Classificators/Helpers/CustomVisionApiSchema.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Globalization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace ImageRecognition.Classificators.Helpers
+{
+// Generated using https://app.quicktype.io/#l=cs&r=json2csharp
+    public partial class CustomVisionPredictionResponse
+    {
+        [JsonProperty("id")]
+        public Guid Id { get; set; }
+
+        [JsonProperty("project")]
+        public Guid Project { get; set; }
+
+        [JsonProperty("iteration")]
+        public Guid Iteration { get; set; }
+
+        [JsonProperty("created")]
+        public DateTimeOffset Created { get; set; }
+
+        [JsonProperty("predictions")]
+        public Prediction[] Predictions { get; set; }
+    }
+
+    public class Prediction
+    {
+        [JsonProperty("probability")]
+        public double Probability { get; set; }
+
+        [JsonProperty("tagId")]
+        public Guid TagId { get; set; }
+
+        [JsonProperty("tagName")]
+        public string TagName { get; set; }
+    }
+
+    public partial class CustomVisionPredictionResponse
+    {
+        public static CustomVisionPredictionResponse FromJson(string json) => JsonConvert.DeserializeObject<CustomVisionPredictionResponse>(json, Converter.Settings);
+    }
+
+    public static class Serialize
```

Comment:
Maybe we should use this class to also serialize our shopping list items? Because right now there's a Json Reader and Writer, we could make them ShoppingListItemReader/Writer and separate the serializing functionality from them? Or is this not relevant on Android? Is Newtonsoft.Json only a PC thing?

Reply:
That's an interesting idea. Though I don't think we can adapt this to be more general unfortunately. This class is dependent on the CustomVision API, so I think it would be best if we leave it as it is for now.